src/database/migrations/0057-alter-base_fields-use-shortcode-as-primary-key.sql

```diff
@@ -0,0 +1,52 @@
+-- 1. Add new columns to tables fkeyd on basefields
+ALTER TABLE base_field_localizations
+ADD COLUMN base_field_short_code varchar;
+ALTER TABLE application_form_fields
+ADD COLUMN base_field_short_code varchar;
+
+-- 2. Populate the new column by joining the base_fields table.
+UPDATE base_field_localizations
+SET base_field_short_code = base_fields.short_code
+FROM base_fields
+WHERE base_field_localizations.base_field_id = base_fields.id;
+UPDATE application_form_fields
+SET base_field_short_code = base_fields.short_code
+FROM base_fields
+WHERE application_form_fields.base_field_id = base_fields.id;
+
+
```

Comment:
Harry, this extra line of whitespace will haunt you until the end of your days because it is in a migration file.